docs-master/Advanced/ProTips.md

```diff
@@ -2,14 +2,14 @@
 
 ## Database viewer
 
-See this discussion: https://github.com/Nozbe/WatermelonDB/issues/710
+[See discussion](https://github.com/Nozbe/WatermelonDB/issues/710)
 
-**Android** - you can use the new database inspector which comes with the Android Studio Beta. https://medium.com/androiddevelopers/database-inspector-9e91aa265316  .
+**Android** - you can use the [new database inspector](https://medium.com/androiddevelopers/database-inspector-9e91aa265316) which comes with the Android Studio Beta.
 
```

Comment:
Database inspector is no longer only in beta branch. (also on newer versions Android Studio 4.2+ it's called `App Inspector` as it allows much more than only checking database)